graders/python/requirements.txt

```diff
@@ -20,8 +20,8 @@ Pygments==2.19.2
 pyarrow==22.0.0
 pygraphviz==1.14
 requests==2.32.5
-scikit-image==0.25.2
-scikit-learn==1.7.2
+scikit-image==0.26.0
```

Comment:
This has some [deprecations](https://github.com/scikit-image/scikit-image/releases). I'm not familiar enough with scikit-image to know if this is significant. I'd appreciate it if someone could double-check it.